docs/sources/panels/inspect-panel.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Inspect a panel"
+type = "docs"
+[menu.docs]
+identifier = "inspect-a-panel"
+parent = "panels"
+weight = 400
++++
+
+# Inspect a panel
+
+> **Note:** This documentation refers to a feature only available in Grafana 7.0 beta.
+
+The panel inspector helps you understand and troubleshoot your panels. You can inspect the raw data for any Grafana panel, export that data to a comma-separated variable (CSV) file, view query requests, and export panel and data JSON.
+
+## Panel inspector UI
+
+The panel inspector displays Inspect: <NameOfPanelBeingInspected> at the top of the pane. Click the arrow in the upper right corner to expand or reduce the pane.
+
+The panel inspector consists of four tabs:
+
+* **Data tab -** Shows the raw data returned by the query with transformations applied. Field options such as overrides and value mappings are not applied.
```

Comment:
Value mappings can be overriden too, it's one of the field config options. I suggest `Field options and field option overrides are not applied`

Reply:
Value mappings are a separate thing in the UI, which is why I mentioned them separately. While they might be just another field config override to Grafana Lab developers, they look like a different thing to the user on the UI side.4

Reply:
oh, you are totally right